forge-gui/res/cardsfolder/upcoming/phyrexian_vindicator.txt

```diff
@@ -0,0 +1,11 @@
+Name:Phyrexian Vindicator
+ManaCost:W W W W
+Types:Creature Phyrexian Horror
+PT:5/5
+K:Flying
+R:Event$ DamageDone | ActiveZones$ Battlefield | ValidTarget$ Card.Self | ReplaceWith$ DBImmediateTrigger | PreventionEffect$ True | AlwaysReplace$ True | Description$ If damage would be dealt to CARDNAME, prevent that damage.
```

Comment:
Should have the full description 
![image](https://user-images.githubusercontent.com/67333662/216825323-38d4659f-9fa6-4dc4-9e66-ed0b78a17fb2.png)